src/ApiClient/AnnotationsClient.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace eLife\ApiClient\ApiClient;
+
+use eLife\ApiClient\ApiClient;
+use GuzzleHttp\Promise\PromiseInterface;
+
+final class AnnotationsClient
+{
+    const TYPE_ANNOTATION_LIST = 'application/vnd.elife.annotation-list+json';
+
+    use ApiClient;
+
+    public function listAnnotations(
+        array $headers,
+        string $by,
+        int $page = 1,
+        int $perPage = 20,
+        bool $descendingOrder = true,
+        bool $updatedSortBy = true
```

Comment:
for consistency with https://github.com/elifesciences/api-client-php/blob/master/src/ApiClient/CoversClient.php#L21 and others, `useDate` should be a string parameter. This choice allows additional values to be passed in the future without having to update the client